kolibri/core/auth/management/commands/deprovision.py

```diff
@@ -60,13 +70,14 @@ def handle_async(self, *args, **options):
         except server.NotRunning:
             pass
 
-        # ensure the user REALLY wants to do this!
-        confirm_or_exit(
-            "Are you sure you wish to deprovision your database? This will DELETE ALL USER DATA!"
-        )
-        confirm_or_exit(
-            "ARE YOU SURE? If you do this, there is no way to recover the user data on this device."
-        )
+        if options["interactive"]:
+            # ensure the user REALLY wants to do this!
+            confirm_or_exit(
+                "Are you sure you wish to deprovision your database? This will DELETE ALL USER DATA!"
+            )
+            confirm_or_exit(
+                "ARE YOU SURE? If you do this, there is no way to recover the user data on this device."
+            )
```

Comment:
We had previously made the explicit choice not to make it that easy to deprovision without warnings, given how destructive it is.

If a one-liner is really needed, we've suggested `yes yes|kolibri manage deprovision` in the past. I guess that doesn't work on Windows, though.

Reply:
Ah, I see it's for the purposes of `call_command`, though -- I wonder about using a nonstandard name like `--accept-data-loss` or something to make it a bit harder to do accidentally.

I'm probably being paranoid. Except if there's anything we've learned from what users do.....

Reply:
Yeah, I only added it because of the python invocation, happy to make it something ridiculous!

Reply:
Throw some unicode for a few different languages in there and we should be good, then!